### PR TITLE
Add possibility of generators customization

### PR DESCRIPTION
**Pull request by nikita-ezhkov**
We would like to use webpack BEM loader, but we are not sure that we want to use [bem-react-core](https://github.com/bem/bem-react-core). So we get an error: `applyDecls` is `undefined`.

### CODE REVIEW

**Comment by nikita-ezhkov** [comment]
@Yeti-or have you got any updates here? We need it

**Comment by nikita-ezhkov** [comment]
@Yeti-or @awinogradov ping

**Comment by nikita-ezhkov** [comment]
@Yeti-or @awinogradov @veged is there anyone how can consider this PR?

File: README.md [hunk 1]
```diff
@@ -82,6 +104,7 @@ module: {
 - __techs__ <Array>: list of techs extensions for require in runtime, `['js']` by default
 - __techMap__ <Object>: mapping of techs to extensions. Example: `{ 'js' : ['react.js', 'react.ts', 'react.es'], 'css' : ['post.css'] }`
 - __langs__ <Array>: list of langs in which resloves '.i18n' tech
+- __generators__ <Object>: customization of code generators by tech. The function when it is provided receive one argument: __files__ with signature `Array<String>`. This is the list of files of the specified technology, got from current import. Examples: `{ js : null }` or ```{ js: (files) => files.map(file => `require('${file.path}')`).join(',\n') }```
```
Comment (veged) [inline README.md]: is it meaningful to do `{ js : null }`?

**Comment by veged** [comment]
@nikita-ezhkov sorry for delay :-( let's do this!

File: README.md (hunk 1 above)
Comment (nikita-ezhkov) [reply README.md]: Another option seems to be changing of default "js" generator what require a major release after that. 

The problem is that JS generator adds call of `.applyDecls()` for the module https://github.com/bem/webpack-bem-loader/blob/master/generators/js.js#L11. So the ability to use this loader independently (without https://github.com/bem/bem-react-core) is missed.

**Comment by veged** [comment]
ok — we will merge it with https://github.com/bem/webpack-bem-loader/pull/58, since there will be new argument in generators

**Comment by nikita-ezhkov** [comment]
@veged it seems that #58 and this one fix the same issues. I had faced with wrong order of files until i did this fix.

**Comment by veged** [comment]
@Yeti-or do you already fix this and #58?

**Pull request closed without merging**